### DISCUSSION TITLE
Stuck at Can't verify until...

### DISCUSSION THREAD

**Discussion by 2022m691** [community/community#52092] [GitHub Education]
Heyo, I'm having trouble applying. I put my full name and wrote a short bio, but it's still saying I have to complete those things. Additionally, I'm uploading a photo that I took with my phone and then mailed it to my laptop, which doesn't have a camera. It's telling me that uploaded files can be easily manipulated. I'll gladly provide any data you may need to get this approval, I have nothing to hide.

**Comment by samus-aran** [comment]
Hi @2022m691 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Regards to the image manipulation although we do prefer live camera images we do accept uploaded files. 

Let us know how you get on 👍
